test/exception_notification/rack_test.rb

```diff
@@ -41,4 +41,22 @@ class RackTest < ActiveSupport::TestCase
     ExceptionNotification::Rack.new(@normal_app, error_grouping: true).call({})
     assert_equal Rails.cache, ExceptionNotifier.error_grouping_cache
   end
+
+  test "should assign @@ignore_if when ignore_if is specified" do
```

Comment:
I was tempted to examine whether new/old API works as expected in the level of this Rack module ... but I'm happy to take these parts away if you feel it is too much.

I'm wondering whether I should test ExceptionNotifier's internal vars from here.